512-byte-vm.asm

```diff
@@ -52,6 +52,14 @@ end_of_print:
     MOV     [BP],AX                     ; get the saved print mode
     JZ      print_next_character        ; at the end of the compressed loop, go back and print normal text
 
+    CALL    serial_port_count
```

Comment:
@lveyde could you add comments to these lines please?

Reply:
Yes, certainly. Will check that tomorrow.